python/samples/documentation_examples/functions_within_prompts.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+
+from service_configurator import add_ai_service
+
+import semantic_kernel as sk
+from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+from semantic_kernel.contents.chat_history import ChatHistory
+from semantic_kernel.core_plugins import ConversationSummaryPlugin
+from semantic_kernel.prompt_template.input_variable import InputVariable
+from semantic_kernel.prompt_template.prompt_template_config import PromptTemplateConfig
+
+
+async def main():
```

Comment:
this one looks 1:1 the same as "configuring_prompts.py", maybe that one should be a simpler version, while this one has the functions...

Reply:
There looks to be 1:1 similarities between python and dotnet as well, so just replicated it here...